Automattic/wp-calypso
# Activity log: use activity endpoint

Add `QueryActivityLog` component.
Use `activity` endpoint.
Cleans up timestamp logic and language.

To test:
1. Visit `http://calypso.localhost:3000/stats/activity/$site` for a Pressable site with activity.
1. Check network requests and watch for traffic at the `activity` endpoint.
1. Ensure network traffic looks OK

Followup:
* `logError` reducer
* reducer tests

Replaces #14795
